core/src/main/java/com/digitalpebble/stormcrawler/Metadata.java

```diff
@@ -243,7 +243,7 @@ public Metadata unlock() {
     }
 
     /** @since 1.16 */
-    private final void checkLockException() {
```

Comment:
Declaring a private method `final` is redundant because you can not override a private function and therefore private functions are already handled like being `final`. This is basically like declaring the methods of an public interface as public.